config.js

```diff
@@ -260,6 +261,36 @@ if (config.get('gcpServiceAccountKey')) {
     config.set('gcpAuthTokenHelper', gcpAuthTokenHelper);
 }
 
+// Load dynamic pathways from JSON file or cloud storage
+const createDynamicPathwayManager = async (config, basePathway) => {
+    const { pathwaysPath } = config.getProperties();
+    const dynamicPathwaysPath = path.join(pathwaysPath, 'dynamic', 'pathways.json');
+
+    const storageConfig = {
```

Comment:
Would this be better as a single env var?  Something like CORTEX_DYNAMIC_PATHWAYS_LOCATION or something. . .  with a format. . .  so like [storage-type]://[config-key1]=[value1];[config-key2]=[value2];...  e.g. For Azure:
azure://ConnectionString=[Your-Azure-Connection-String];ContainerName=cortexdynamicpathways
For AWS:
aws://AccessKeyId=[Your-Access-Key];SecretAccessKey=[Your-Secret-Key];Region=[Your-Region];BucketName=cortexdynamicpathways
For local storage:
local://FilePath=/path/to/dynamic/pathways

With a simple parser?  So that keeps us from env var hell when they're all related to one feature.